### ISSUE TITLE
Webhook posts are failing

### ISSUE THREAD

**Issue by bbarton** [j03b/webhook-discord#59]
Hi,

My app spontaneously stopped posting to Discord several hours ago (no code changes on my part, tested on multiple envs).

Was running 3.6.0 and updated to 3.7.4 to no avail (however, reverting to 3.5.1 seems to work).

```
async function test() {
  try {
    webhook = require("webhook-discord");
    Hook = new webhook.Webhook(webhook_url);
    msg = new webhook.MessageBuilder().setName("Async").setColor("#aabbcc").setText("This is my webhook!");
    await Hook.send(msg)
  } catch (err) {
    console.log("Error:", err);
  }
}

```

Produces this error:

```
(node:38467) UnhandledPromiseRejectionWarning: Error: 400 Bad Request
    at /Users/me/Desktop/myproject/node_modules/snekfetch/src/index.js:195:23
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
(node:38467) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:38467) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Any idea what might be causing this? Thanks!

**Comment by isaacaddis** [comment]
Same here, they've suddenly stopped working!

**Comment by NetWalkerGit** [comment]
I have same problem as above.

**Comment by anishanne** [comment]
I've downgraded to 3.5.1 and I'm getting the same error.

**Comment by 4rlon** [comment]
same

**Comment by j03b** [comment]
Interesting...

**Comment by j03b** [comment]
Fixed in webhook-discord@3.7.5

Thanks @anishanne for the contribution.
